Source/gs/GSH_OpenGL/GSH_OpenGL.cpp

```diff
@@ -770,6 +770,13 @@ void CGSH_OpenGL::SetupBlendingFunction(uint64 alphaReg)
 		glBlendColor(0, 0, 0, static_cast<float>(alpha.nFix) / 128.0f);
 		glBlendFuncSeparate(GL_ONE_MINUS_CONSTANT_ALPHA, GL_CONSTANT_ALPHA, GL_ONE, GL_ZERO);
 	}
+	else if((alpha.nA == 1) && (alpha.nB == 0) && (alpha.nC == 2) && (alpha.nD == 1))
```

Comment:
Please use `ALPHABLEND_ABD_*` and `ALPHABLEND_C_*` constants instead of numbers.

Reply:
Done